tests/bin/initialize.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/bash
 set -e
 
-npm run env run tests-wordpress "chmod -c ugo+w /var/www/html"
-npm run env run tests-cli "wp rewrite structure '/%postname%/' --hard"
+npx wp-env run tests-wordpress chmod -c ugo+w /var/www/html
```

Comment:
We needed to change from `npm run env run` to `npx wp-env run` as after removing the quotes, flags like `-c ugo+w` and `--slug=second` were being omitted while passing the command to wp-env. My best guess is that those flags were passed to npm instead.

Reply:
@kirtangajjar As this shell script is triggered by the npm after script, I think we can directly use `wp-env` and avoid using `npx`.